db/migrate/20220616131226_add_unique_index_to_statement_line_items.rb

```diff
@@ -0,0 +1,11 @@
+class AddUniqueIndexToStatementLineItems < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured do
+      remove_index :statement_line_items, :statement_id
```

Comment:
are we sure we want to remove this? if we want to grab all the line items for a particular statement it now needs to scan the whole table?

Reply:
this is covered in those indices https://github.com/DFE-Digital/early-careers-framework/pull/2225/files#diff-8d6085c3e87baff35cf7da33c61ceb84b635522fe5b34f0b4e518a03e9c7478eR9-R10